Presentation.Web/app/components/data-processing/tabs/data-processing-registration-tab-main.view.html

```diff
@@ -167,43 +167,61 @@ <h4 data-element-type="dpaMainDetailHeader">{{vm.headerName}}</h4>
-                                    <i class="glyphicon glyphicon-trash"></i>
-                                </button>
-                            </td>
-                        </tr>
+                            <tr ng-repeat="dataProcessor in vm.subDataProcessors.selectedElements" data-element-type="data-processing_sub-data-processors">
+                                <td>{{::dataProcessor.name }}</td>
+                                <td class="noRightBorder">{{::dataProcessor.cvrNumber }}</td>
+                                <td class="text-right noLeftBorder" style="width: 20px;">
+                                    <button type="button"
+                                            class="btn btn-danger btn-xs"
+                                            data-confirmed-click="vm.subDataProcessors.removeItemRequested({{::dataProcessor}})"
+                                            data-confirm-click="Er du sikker på at du vil slette tilknytningen?"
+                                            ng-if="vm.subDataProcessors.allowRemoval">
+                                        <i class="glyphicon glyphicon-trash"></i>
+                                    </button>
+                                </td>
+                            </tr>
                         </tbody>
                         <tfoot ng-if="vm.subDataProcessors.allowAddition" class="element-selection-list_footer--padding">
-                        <tr>
-                            <td colspan="3" style="padding: 0">
-                                <single-select-select2 data-placeholder="Vælg Underdatabehandler"
-                                                       id="sub-data-processor_select-new"
-                                                       ng-model="vm.subDataProcessors.newItemSelectionConfig"
-                                                       ng-disabled="!vm.subDataProcessors.allowAddition" />
-                            </td>
-                        </tr>
+                            <tr>
+                                <td colspan="3" style="padding: 0">
+                                    <single-select-select2 data-placeholder="Vælg Underdatabehandler"
+                                                           id="sub-data-processor_select-new"
+                                                           ng-model="vm.subDataProcessors.newItemSelectionConfig"
+                                                           ng-disabled="!vm.subDataProcessors.allowAddition" />
+                                </td>
+                            </tr>
                         </tfoot>
                     </table>
                 </div>
+                <label class="control-label small">Status</label>
+                <!-- Show the simple version of the status if there are no details to explain-->
+                <div class="form-group object-status-active-summary" ng-if="vm.validationStatus.valid">
```

Comment:
might need an additional value to prevent nulle reference on vm.validationStatus.valid OR just additional component in the ng-if